### DISCUSSION TITLE
Swap Always Full But No Issues. Is it fine? Raspi 4B 4GB

### DISCUSSION THREAD

**Discussion by AlaskanAstro** [aaronwmorris/indi-allsky#1367] [Q&A]
I haven't done any tweaking to the system yet, just get a >90% Swap Message every time I open up the webapage. My timelapses and such are all auto-generating fine and I don't notice any lag when flipping through pages. Is there any reason to be worried about swap basically always living at 100%?

**Comment by aaronwmorris** [comment]
Its probably due to the default swappiness setting and there is only 100MB of swap by default on a system.  Reducing swappiness to 1 should prevent the system from swapping unnecessarily.

https://github.com/aaronwmorris/indi-allsky/wiki/Memory-Optimizations

**Reply by AlaskanAstro** [reply]
Thanks, just did it. We'll see if anything changes.

**Comment by AlaskanAstro** [comment]
Just a follow up, could there be an option to disable that warning message if it doesn't really cause any issues? Or can I set the threshold to 101% or something?

**Reply by aaronwmorris** [reply]
Yeah, just set the threshold to 101 and it will disable.  The only other option is to increase the amount of swap to something like 256 or 512 and it will be less likely to hit the threshold.
